internal/config/cgmgr/cgmgr_linux.go

```diff
@@ -59,13 +59,23 @@ type CgroupManager interface {
 	// PopulateContainerCgroupStats fills the stats object with information from the cgroup found
 	// given a cgroup parent and container ID.
 	PopulateContainerCgroupStats(sbParent, containerID string, stats *types.ContainerStats) error
+	// ContainerCgroupManager takes the cgroup parent, and container ID.
+	// It returns the raw libcontainer cgroup manager for that container.
+	ContainerCgroupManager(sbParent, containerID string) (libctr.Manager, error)
+	// RemoveCtrCgManager removes the cgroup manager for the container
+	RemoveCtrCgManager(containerID string)
```

Comment:
I think we should keep the names consistent. RemoveContainer* and RemoveSandbox* below